src/config.cpp

```diff
@@ -370,7 +387,15 @@ RecompPort::Config::Config(const char* path) {
         toml::node_view manual_functions_data = input_data["manual_funcs"];
         if (manual_functions_data.is_array()) {
             const toml::array* array = manual_functions_data.as_array();
-            get_manual_funcs(array);
+            manual_functions = get_manual_funcs(array);
+        }
+
+        std::optional<std::string> output_binary_path_opt = input_data["output_binary_path"].value<std::string>();
+        if (output_binary_path_opt.has_value()) {
+            output_binary_path = concat_if_not_empty(basedir, output_binary_path_opt.value());
```

Comment:
super minor: but given how this method is being used (return base path with child if not empty, otherwise return base path) it feels like a better name might be `append/concat_subpath_if_not_empty` (if you want the logic to be explicit in the name) or even just a general `resolve_path` - implementation details not necessarily being exposed in the name.

Reply:
I agree that the name isn't the greatest, but `resolve_path` implies it more than just concatenating two paths together (like resolving symlinks for example), so I'll leave it like this for now.